linkerd/smi_conformance.go

```diff
@@ -0,0 +1,49 @@
+package linkerd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/layer5io/gokit/smi"
+	"github.com/layer5io/meshery-linkerd/meshes"
+)
+
+func (iClient *Client) runConformanceTest(id string, name string, version string) error {
+
```

Comment:
I have already add `golangci-lint` to check these issues, but we need to merge the PR first

Reply:
@Aisuko  I have merged that. Lets check this now?